atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -207,7 +207,7 @@ def run(self):
             self.log.debug("annotations: %r", annotations)
             raise
 
-        labels = self.make_labels()
+        labels = self.make_labels(metadata)
```

Comment:
`store_metadata` doesn't read existing metadata (e.g. `koji-task-id` label) and returns an incomplete dict of labels - it would only contain a new labels added - `koji-build-id`.

Since there is no easy way to re-read build labels here, the plugin returns a full list of labels